docs/api-docs/storefront/widgets/widget-builder.md

```diff
@@ -110,6 +112,46 @@ re you sure? No
 
 If you need to reset the configurations, run the `widget-builder init` command to overwrite the existing assignments.
 
+## Building widgets
+
+To start Widget Builder locally, open the directory containing your widget template files and run the following command:
+
+```shell
+widget-builder start [path to widget template]
+```
+Your default browser should open automatically on port 8080.
+
+### Starter template
+
+To jump start widget development, Widget Builder CLI provides a blank template with the appropriate files needed to start building widgets. To use the starter template, run the following command:
```

Comment:
To jump-start widget development, Widget Builder CLI provides a blank template with the appropriate files needed to start building widgets. To use the starter template, run the following command: